package.json

```diff
@@ -53,7 +53,10 @@
     "react-native-web": "0.17.1"
   },
   "devDependencies": {
-    "@babel/core": "^7.12.9"
+    "@babel/core": "^7.12.9",
+    "@types/react": "~17.0.21",
+    "@types/react-native": "~0.64.12",
+    "typescript": "~4.3.5"
```

Comment:
ajout de prettier?

Reply:
dans une autre PR pour pouvoir merger rapidement

Reply:
Why not using the last version of typescript?